### DISCUSSION TITLE
Why is there no max-len rule for TypeScript?

### DISCUSSION THREAD

**Discussion by wesleyschlenker** [eslint-stylistic/eslint-stylistic#555] [Q&A]
My org is migrating to using TS, and we noticed there's no max-len enforcement for it, no matter how we tweak our rules settings. It appears that this rule is only available in the JS plugin. Is there a reason for this? Is there a plan to implement that rule? Am I missing something, or must we use a different formatting tool to receive this functionality?

**Comment by 9romise** [comment]
We recommend using [@stylistic/eslint-plugin](https://eslint.style/packages/default) directly, and in future versions, both `js` and `ts` versions will be [merged into one package](https://github.com/eslint-stylistic/eslint-stylistic/issues/482).
If you encounter any problems when using this rule, please create an issue.
